helm/loki-stack-app/charts/loki/values.yaml

```diff
@@ -67,7 +67,7 @@ config:
 
 image:
   repository: grafana/loki
```

Comment:
maybe you wanna also add a default registry like in https://github.com/giantswarm/coredns-app/blob/master/helm/coredns-app/values.yaml#L23

Reply:
ah, just saw you template registry on the higher level, disregard here and below